docs/architecture/commitlogs.md

```diff
@@ -46,3 +46,7 @@ CommitLogMetadata {
 ### Garbage Collected
 
 Commit logs are garbage collected after all blocks within the retention period in which data inside the commit logs could be applicable have already been flushed to disk as immutable compressed filesets.
+
+### Compaction
```

Comment:
Could you add a quick overview section to the beginning of the commit log page. Just detail that it's a WAL equivalent that is uncompressed, one per database, blah blah...

Reply:
done